### ISSUE TITLE
Problem : Extra column type `:city` defined in COLUMN_TYPE

### ISSUE THREAD

**Issue by pywugate** [BoZenKhaa/ReverseGeocode.jl#12]
### Problem : extra column type `:city => String` defined in `COLUMN_TYPE` 
  
There is no "city" field in original data at version 2024-05-02 04:23.
So `:city => String` in `COLUMN_TYPE = OrderedDict()` causes many missing during reading CSV and destroys the csv table.

I didn't see directly usage of `:city` in the code,
only `rename(data, :name => :city)` , using `:name` instead `:city` column

I think `:city => String` in `COLUMN_TYPE = OrderedDict()` can be put at the end or even be deleted

**Comment by BoZenKhaa** [comment]
Yeah, I want the column names to reflect the names in the [geoname table](https://download.geonames.org/export/dump/), I will remove `:city` and replace it with `:name` for the next version.
